react/src/components/Files/FileListing.tsx

```diff
@@ -290,29 +303,23 @@ export const FileListing: React.FC<FileListingProps> = ({
         </FormProvider>
       </div>
       <div className={`${styles['file-browser']}`}>
-        {hasError ? (
-          <Flex align="center" justify="center" style={{ height: '100%' }}>
-            There was an error loading the system.
-          </Flex>
```

Comment:
Is there a way for us to still show an error? For example, error 403 is "Permission denied", which might be more useful to show than "Nothing to show"

Reply:
I added this error back in. Now, when there is a listing error, it will show an error message instead of `Nothing to Show`